docs/UserGuide.md

```diff
@@ -539,22 +546,26 @@ Command Line Interface. Refers to computer programs which require you to type to
 ### GUI
 Graphical User Interface. Refers to computer programs with a visual window you can interact with directly.
 
-### Patient List View
-The main part of HealthSync where the list of all your patients are displayed.
+### [Menu Sidebar](#menu-sidebar)
+The Menu Sidebar is the area on the left of the Patient List View. It contains buttons that allow you to perform actions on HealthSync. Currently, only the `User Guide` and `Exit` buttons are functional. The buttons that are in grey and not pressable in the Menu Sidebar are currently not functional and will be implemented in a future version of HealthSync.
 
-### Output Box
-A small box right below the Patient List View where HealthSync will provide any feedback it has on the instructions
-it has performed.
+### [Patient List View](#patient-list-view)
+The main part of HealthSync where the list of all your patients are displayed. This is where you can view patient profiles in the form of cards, and perform actions on them. When actions are performed, the Patient List View will be updated to reflect the changes/results.
```

Comment:
```suggestion
The main part of HealthSync where the list of all your patients are displayed. This is where you can view patient profiles in list form, and perform actions on them. When actions are performed, the Patient List View will be updated to reflect the changes/results.
```

Reply:
Have some concerns that 'cards' may be a bit technical, as in Person Cards, and readers may not understand what that means. Perhaps keeping the phrasing standardised to list would be clearer?

Reply:
I think cards could be interpreted in a technical way. However, I also think that a 'name card' or 'contact card' is a widespread enough concept that we can potentially get away with keeping it as this form.

@WeeeHung we will leave it to your discretion if you want to change it. Let us know if we can merge by resolving this conversation.